systems/data-plan/base-rate/pkg/db/rate_repo.go

```diff
@@ -20,18 +22,19 @@ func NewBaseRateRepo(db sql.Db) *baseRateRepo {
 	}
 }
 
-func (u *baseRateRepo) GetBaseRate(rateId uint64) (*Rate, error) {
+func (u *baseRateRepo) GetBaseRate(rateID uuid.UUID) (*Rate, error) {
 	rate := &Rate{}
-	result := u.Db.GetGormDb().First(rate, "Id=?", rateId)
+	result := u.Db.GetGormDb().First(rate, "rate_id=?", rateID)
```

Comment:
In api gateway this in uint64 here its UUID

Reply:
I have updated it to UUID in all instances.